youtube_dl/extractor/francetv.py

```diff
@@ -166,6 +166,21 @@ def sign(manifest_url, manifest_id):
                         'url': video_url,
                         'format_id': format_id,
                     })
+
+        if not formats:
+            fallback_info = self._download_json(
+                'https://player.webservices.francetelevisions.fr/v1/videos/%s' % video_id,
+                video_id, 'Downloading fallback video JSON', query={
+                    'device_type': 'desktop',
+                    'browser': 'ytdl',
+                })
+
+            video_url = fallback_info['video']['url']
+            format_id = fallback_info['video']['format']
+
+            formats.extend(self._extract_mpd_formats(
+                sign(video_url, format_id), video_id, mpd_id='dash', fatal=False))
```

Comment:
reuse [the code that parses the old formats](https://github.com/ytdl-org/youtube-dl/blob/aa613ef7e1efe9f799a1209659f8d9d01e3de221/youtube_dl/extractor/francetv.py#L135-L168).